src/test/java/ladder/domain/ladder/LadderTest.java

```diff
@@ -0,0 +1,48 @@
+package ladder.domain.ladder;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.SoftAssertions.assertSoftly;
+
+import java.util.List;
+import ladder.domain.ladder.generator.BooleanGenerator;
+import ladder.domain.ladder.generator.RandomBooleanGenerator;
+import ladder.mock.MockBooleanGenerator;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class LadderTest {
+
+    @Test
+    @DisplayName("사다리 생성자 테스트")
+    void testConstruct() {
+        int playerCount = 4;
+        LadderHeight ladderHeight = new LadderHeight(5);
+        BooleanGenerator booleanGenerator = new RandomBooleanGenerator();
+        Ladder ladder = new Ladder(playerCount, ladderHeight, booleanGenerator);
+
+        assertThat(ladder).extracting("playerCount")
+                .isEqualTo(4);
+        assertThat(ladder).extracting("ladderHeight")
+                .isEqualTo(ladderHeight);
+    }
+
+    @Test
+    @DisplayName("사다리를 생성한다.")
+    void testGenerateLadder() {
+        int playerCount = 4;
+        LadderHeight ladderHeight = new LadderHeight(3);
+        List<Boolean> rungExist = List.of(true, false, true, false, true, false, true, false, false);
```

Comment:
저라면 아래처럼 했을거 같아요!
```suggestion
        List<Boolean> rungExist = List.of(
        true, false, true, // EXIST, EMPTY, EXIST
         false, true, false, // EMPTY, EXIST, EMPTY
          true, false, false // EXIST, EMPTY, EMPTY
          );
```
테스트는 남이 본다는 시각으로 가능한 친절하게 짜는게 좋습니다. 나만 유지보수 가능한 테스트는 제가 휴가 갔다오고나면 삭제되어 있습니다 (결제 장애가 나서 당장 배포해야되는데 의미를 모르겠는 테스트가 배포를 막고 있다는 상상을 해보시면 좋습니다)